legal-api/src/legal_api/services/authz.py

```diff
@@ -314,6 +315,13 @@ def get_allowable_filings_dict():
                             'business': [BusinessBlocker.DEFAULT]
                         }
                     }
+                },
+                'noticeOfWithdrawal': {
+                    'legalTypes': ['BC', 'BEN', 'CC', 'ULC', 'C', 'CBEN', 'CUL', 'CCC'],
+                    'blockerChecks': {
+                        'business': [BusinessBlocker.BUSINESS_FROZEN,
+                                     BusinessBlocker.NO_FUTURE_EFFECTIVE_FILING]
```

Comment:
There's a scenario that I think you should think of...A business has a FED filing and not in frozen status but it also has a draft NoW filing, in this case, it should be not allowed to start another NoW filing again.